app/assets/stylesheets/application.bootstrap.scss

```diff
@@ -337,4 +337,10 @@ fieldset.form-fieldset legend {
   input:not(:checked) + label  {
     @extend .btn-outline-primary;
   }
-}
\ No newline at end of file
+}
+
+.card-text {
+  a {
+    text-decoration: underline !important;
```

Comment:
https://dequeuniversity.com/class/visual-design/color/distinguish-links-from-text

Reply:
@aaron-collier I can't get to this page. Can you say more about what is happening? It seems surprising that we should be overriding `<a>`

Reply:
They are reporting that the color contrast between the link text and the non-link text is not high enough. One suggestion is to underline links, which felt a bit easier than shifting away from standard bootstrap colors.

Reply:
This seems like it should be a component library issue. cc @thatbudakguy @astridu

Reply:
I removed this override in favor of giving the link the `card-link` class to keep it within the bootstrap idiom here. Not sure if that resolves the contrast issue however.

Reply:
While I agree these should be component library or design issues, I'm unclear if it's OK for us to wait for that. If it's not fixed before the app is in use, but it's on the horizon for being fixed in the component library, is that OK?